docs/conf.py

```diff
@@ -20,10 +20,34 @@
 import os
 
 import sys
+import yaml
+from datetime import datetime
+
 sys.path.insert(0, os.path.abspath('..'))
 
 from patroni.version import __version__
 
+project_root = os.path.abspath(os.path.join(os.path.dirname(__file__), '..'))
+
+with open(os.path.join(project_root, 'docs', 'metadata.yaml')) as meta_fh:
+    metadata = yaml.safe_load(meta_fh)
+
+module_dir = os.path.abspath(os.path.join(project_root, metadata.get('source_root', '.')))
+sys.path.insert(0, module_dir)
+
+# -- Project information -----------------------------------------------------
+
+project = metadata['project']
+author = metadata['author'][0]
+try:
+    copyright_year = datetime.strptime(metadata['date'], '%d/%m/%Y').year
+except TypeError:
+    copyright_year = metadata['date'].year
+copyright = '{}, {}'.format(copyright_year, author)
```

Comment:
It all just to parse the `docs/metadata.yaml` file, which is not used anywhere else?
Doesn't look like a good improvement comparing to having just [3 lines](https://github.com/zalando/patroni/pull/2699/files#diff-85933aa74a2d66c3e4dcdf7a9ad8397f5a7971080d34ef1108296a7c6b69e7e3L55-L57) in the `conf.py`.

Reply:
No problem. I'll remove the external metadata load.